Preprocessing/StandardScaler.ecl

```diff
@@ -0,0 +1,95 @@
+/*###################################################################################
+## HPCC SYSTEMS software Copyright (C) 2020 - 2021 HPCC Systems.  All rights reserved.
+##################################################################################### */
+
+IMPORT $.^ as ML_Core;
+
+NumericField := ML_Core.Types.NumericField;
+KeyLayout := ML_Core.Preprocessing.Types.StandardScaler.KeyLayout;
+
+/**
+  * Transform the input data to a standard normally distributed data with zero mean and
```

Comment:
Should read:  Standardize the data by mapping to zero mean and standard deviation of 1.0.